bundles/org.openhab.binding.juicenet/src/main/java/org/openhab/binding/juicenet/internal/handler/JuiceNetDeviceHandler.java

```diff
@@ -255,6 +255,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
                     break;
                 }
+                case CHANNEL_OVERRIDE: {
+                    String state = ((OnOffType) command).toString();
+                    if (state.equals("OFF")) {
```

Comment:
Please first check that command is an instance of `OnOffType`.
Then compare to OFF without the need to convert to a string.